docs/docs/faq.mdx

```diff
@@ -154,6 +154,55 @@ When you create an SDK connection, it is linked to a specific environment. You c
 
 Throughout the GrowthBook application, we randomly celebrate key milestones like launching experiments with on-screen confetti. If you'd like to disable this, you can click on your avatar in the top right corner and select "Edit Profile". From there, you can disable the toggle for "Allow Celebrations". Please note this is persisted in your browser's local storage, so if you clear your browser's local storage, you will need to disable this again.
 
+## How do I make my own identifier?
+
+There are cases when using feature flags client side where the 3rd party identifiers used for assignment will be slow to
+load, and may cause flickering as some of the DOM rerenders. In these cases, generating your
+own identifier will make sure that features are assigned correctly when GrowthBook loads. This id that is generated will
+typically align one to one with the other identifiers, and does not need to be passed outside the SDK.
```

Comment:
Might we want to mention that this is for a logged-out identifier, and that if the user logs in to another device they will count as a different user.  Hence it would not match up one to one with a logged-in identifier.